pkg/scheduler/framework/runtime/framework_test.go

```diff
@@ -2236,7 +2259,7 @@ func TestRecordingMetrics(t *testing.T) {
 				SchedulerName: testProfileName,
 				Plugins:       plugins,
 			}
-			f, err := newFrameworkWithQueueSortAndBind(r, profile, withMetricsRecorder(recorder))
+			f, err := newFrameworkWithQueueSortAndBind(r, profile, stopCh, withMetricsRecorder(recorder))
```

Comment:
better to move `defer close(stopCh)` just after `stopCh := make(chan struct{})`. 
`stopCh` won't be closed when `if err != nil {}`

Reply:
Good catch, but we manually `close(stopCh)` on purpose below, so I prefer to add `close(stopCh)` when `err != nil`.

Reply:
👍